src/utils.h

```diff
@@ -442,4 +444,19 @@ const char *ibpi2str(enum ibpi_pattern ibpi);
 int get_value_for_ibpi(enum ibpi_pattern ibpi, const struct ibpi_value ibpi_values[]);
 enum ibpi_pattern get_ibpi_for_value(const int value, const struct ibpi_value ibpi_values[]);
 
+enum output_format {
```

Comment:
json output is used only for ledctl. It should be implemented only for ledctl then.
If we can change controller printing then it should be possible. Let me know if you see other blockers.

Reply:
Removed all changes to utils